test/e2e/storage/persistent_volumes-local.go

```diff
@@ -53,98 +60,309 @@ const (
 	// TODO: This may not be available/writable on all images.
 	hostBase      = "/tmp"
 	containerBase = "/myvol"
-	testFile      = "test-file"
-	testContents  = "testdata"
-	testSC        = "local-test-storagclass"
+	discoveryDir  = "disks" // 'hostBase + discoveryDir' is the path for volume discovery.
```

Comment:
super nit: more typical to place the comment before the field rather than inline...